codeforboston/maple
# Bootstrap migration for About Pages

# Summary

This PR covers the bootstrap migration for the About Pages components: #1417 

This finishes up the work started in https://github.com/codeforboston/maple/pull/1439 - that PR has been abandoned in favor of this one per discussion at the last hack night.

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.
- [na] I've made pages responsive and look good on mobile.

# Screenshots

N/A (Looks identical to prod)

# Known issues

N/A

# Steps to test/reproduce

1. Go to the about pages
1. Verify that they look identical to production
